packages/ra-ui-materialui/src/button/CreateButton.js

```diff
@@ -30,16 +29,17 @@ const styles = theme =>
         floatingLink: {
             color: 'inherit',
         },
-    });
+    })
+);
 
 const CreateButton = ({
     basePath = '',
     className,
-    classes = {},
     label = 'ra.action.create',
     icon = <ContentAdd />,
     ...rest
 }) => {
+    const classes = useStyles();
```

Comment:
As we want to let developers override default styles, react-admin components still accept a `classes` prop that theu pass to `useStyles`. Here is an example: https://github.com/marmelab/react-admin/blob/f8a8b9f0b42f6bf88d11e4b1f6d06dbde95a934a/packages/ra-ui-materialui/src/button/BulkDeleteWithConfirmButton.js#L42-L54

Reply:
Ah, I see. Will make the changes.

Reply:
I've added the the ability to override the `classes` prop for the components.